src/components/Group/index.vue

```diff
@@ -1,5 +1,7 @@
 <template>
-  <slot />
+  <div>
```

Comment:
linting error complaining about having `slot` as the root element, so wrapped in a div